src/mixins/java/org/spongepowered/common/mixin/api/minecraft/world/effect/MobEffectInstanceMixin_API.java

```diff
@@ -87,6 +93,7 @@ public DataContainer toContainer() {
                 .set(Constants.Item.Potions.POTION_DURATION, this.duration)
                 .set(Constants.Item.Potions.POTION_AMPLIFIER, this.amplifier)
                 .set(Constants.Item.Potions.POTION_AMBIANCE, this.ambient)
-                .set(Constants.Item.Potions.POTION_SHOWS_PARTICLES, this.visible);
+                .set(Constants.Item.Potions.POTION_SHOWS_PARTICLES, this.visible)
+                .set(Constants.Item.Potions.POTION_SHOWS_ICON, this.showIcon);
```

Comment:
Do we need to bump the content version ?

Reply:
@gabizou Am I right in thinking that we don't need to bump the version if we're just adding a field, or is it good practice to anyway?

Reply:
It's actually required to bump the version because it's a non-optional field whereas previous potions will not have the icon available.